image_provisioner/playbooks/roles/setup-image/tasks/main.yaml

```diff
@@ -14,25 +14,31 @@
     dest: "{{ tmp_dir }}"
   when: base_image.stat.exists == False
 
-- name: add empty space to disk image as specified in the inventory
-  command: qemu-img resize {{ base_image_location }}  +{{ extend_disk_gb }}G
+- block:
```

Comment:
I was testing it, Sorry I didn't push some commits. I will make the final changes and let you know.